src/include/verbose.h

```diff
@@ -134,6 +134,20 @@ struct __wt_verbose_multi_category {
 #define WT_VERBOSE_ISSET(session, category) \
     WT_VERBOSE_LEVEL_ISSET(session, category, WT_VERBOSE_LEVEL_DEFAULT)
 
+/* Set the verbose level and save the previous value. */
+#define WT_VERBOSE_SAVE(session, verbose_tmp, category, level)   \
+    do {                                                         \
+        verbose_tmp[category] = S2C(session)->verbose[category]; \
+        WT_SET_VERBOSE_LEVEL(session, category, level);          \
+    } while (0)
+
+/* Restore the original level  */
+#define WT_VERBOSE_RESTORE(session, verbose_tmp, category)              \
+    do {                                                                \
+        verbose_tmp[category] = S2C(session)->verbose[category];        \
```

Comment:
Apologies @y123456yz I don't think I explained this properly last time. The changes in `WT_VERBOSE_SAVE` look good now, but `WT_VERBOSE_RESTORE` doesn't need to read the current verbosity level and we can delete this line.
```suggestion
```
Could we also update `verbose_tmp` to `verbose_orig_level` like we did in `__gen_drain_callback(`?